src/app/app.module.ts

```diff
@@ -92,6 +97,9 @@ const routes: Routes = [
         component: HeaderToolbarComponent
       },
       { path: 'domainregistration', component: DomainRegisterComponent},
+      { path: 'identities', component: ProfilesComponent},
```

Comment:
If we settled on "Identities" then `ProfilesComponent` should probably be renamed.

Reply:
Its from identity profile =p 
it was initially profile until identity seemed better